indico/cli/util.py

```diff
@@ -112,3 +112,7 @@ def get_usage(self, ctx):
 
     def get_params(self, ctx):
         return self._impl.get_params(ctx)
+
+
+def format_event(event):
```

Comment:
I'd keep this in event.py; it's only needed for event-related CLIs any cli/util,py is mostly for generic cli utils (see the comment in line 19+)